astropy/utils/data_info.py

```diff
@@ -490,17 +490,17 @@ class = Column
             info["name"] = name
 
         options = option if isinstance(option, (list, tuple)) else [option]
-        for option in options:
-            if isinstance(option, str):
-                if hasattr(self, "info_summary_" + option):
-                    option = getattr(self, "info_summary_" + option)
+        for opt in options:
```

Comment:
This is a place where ruff just makes code less clear: in the line above, `option` has effectively been made irrelevant, and it is nice to have `for singular in plural` -- and now the someone reading the code has to deal with an abbreviation (less clear) and more verbose exception.

If we go this route, maybe at least get rid of the abbreviation by appending an underscore, and/or do
```
option = option if isinstance(option, (list, tuple)) else [option]
for option_ in option:
    ...
```

p.s. Not a big enough deal to stop this, but it explains my fatigue with the stylistic changes...

Reply:
For the record, this rule (PLR1704) is from pylint (which I personally feel is generally noisy). Though the refactor is manual and I don't want to hurt readability. I'll use `option_` as suggested.